components/collection/drop/GenerativePreview.vue

```diff
@@ -116,7 +116,7 @@ const { imageDataPayload, imageDataLoaded } = useGenerativeIframeData()
 const { start: startTimer } = useTimeoutFn(() => {
   // quick fix: ensure that even if the completed event is not received, the loading state of the drop can be cleared
   // only applicable if the drop is old one that missing`kodahash/render/completed` event
-  if (!props.mintCountAvailable && !imageDataLoaded.value) {
+  if (props.mintCountAvailable && !imageDataLoaded.value) {
     isLoading.value = false
     emit('generation:end')
   }
```

Comment:
This block of code is only for the old drop that does not emit `kodahash/render/completed` event. https://github.com/vikiival/kodahash/blob/28a42b820d639babf8453dc8b477a8ab9218660a/public/sketch.js#L57

Reply:
@vikiival Should we fix the html resource(without `kodahash/render/completed` event) from the drop? Or just hack on the frontend?

Reply:
What are the drops that needs to be updated?

Reply:
We revert these [changes](https://github.com/kodadot/nft-gallery/pull/9586/files) then if we fix that on the drop side?

Reply:
Sure i can update reflrection